### ISSUE TITLE
aria-live attribute has an invalid value for content on Offices and Services page

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#678]
`aria-live` attribute is set to `true` for content on Offices and Services page, permitted values for it are `assertive`, `polite` or `off`. 

Siteimprove page report: https://my2.siteimprove.com/Inspector/7333/A11Y/Page?pageId=72160253158&impmd=D97548790CE293BDC08F6D9B84D3E607&conf=a+aa+aria+si&issue=cantTell+failed&wcag=twopointone#/sia-r19/failed/UFZjTU54UU4=;MThhMWQ2ZDM=

Relevant MDN article: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live
